tests/builder/Data/ByteString/Builder/Tests.hs

```diff
@@ -1,3 +1,7 @@
+{-# LANGUAGE BlockArguments   #-}
```

Comment:
As witnessed by CI, we still support GHCs too old to know about `BlockArguments`.